src/sdk/modules/smartSessionsValidator/decorators/trustAttesters.ts

```diff
@@ -30,6 +30,78 @@ export type TrustAttestersParameters<
   threshold?: number
 }
 
+/**
+ * Gets the action data for trusting attesters in the smart session validator's registry.
+ * 
+ * This function prepares the calldata and execution parameters needed to trust attesters
+ * in the registry contract. It encodes the function call with the provided attesters and threshold.
+ *
+ * @param params - The parameters object
+ * @param params.chainId - The ID of the blockchain network
+ * @param params.trustAttestersInfo - Configuration for trusting attesters including addresses and threshold
+ * @param params.client - The public client used to interact with the blockchain
+ * @returns A promise that resolves to an Execution object containing the target, value and calldata,
+ *          or an Error if the operation fails
+ *
+ * @example
+ * ```ts
+ * const action = await getTrustAttestersAction({
+ *   chainId: 1,
+ *   trustAttestersInfo: {
+ *     attesters: ['0x...', '0x...'],
+ *     threshold: 2
+ *   },
+ *   client: publicClient
+ * });
+ * ```
+ */
+// Review if we need this: as it is available in Rhinestone module sdk as well!
```

Comment:
check

Reply:
Here also can use rhinestone's "getTrustAttestersAction"

Reply:
ok I can do that.

Reply:
done. check